tests/unit/plugins/test_kernel.py

```diff
@@ -742,11 +742,22 @@ def fake_output(*args, **kwargs):
                            plugin.installdir, 'lib', 'firmware'))])
         ])
 
+        class _check_env:
+            def __init__(self, test):
+                self.test = test
+
+            def __eq__(self, other):
+                self.test.assertThat(other, Contains('PATH'))
+                paths = other['PATH'].split(':')
+                self.test.assertThat(paths, Contains('/sbin'))
+                return True
+
         self.assertThat(self.run_output_mock.call_count, Equals(1))
         self.run_output_mock.assert_has_calls([
             mock.call([
                 'modprobe', '-n', '--show-depends', '-d',
-                plugin.installdir, '-S', '4.4.2', 'my-fake-module'])])
+                plugin.installdir, '-S', '4.4.2', 'my-fake-module'],
+                env=_check_env(self))])
```

Comment:
I find passing an object here that calls assertions rather confusing, compared to the `os.environ.copy(); env['PATH']=...; env=env`.

More importantly, there should probably be two separate tests with a fake environment that has and hasn't got sbin in the PATH respectively, so that the outcome doesn't depend on the host?

Reply:
How do you suggest verifying that `/sbin` would put on the PATH? This has always been my approach, do you know a better one? Regarding depending on the host, that's a good call, but it doesn't require another test, I can just set the PATH for this one.

Reply:
> I can just set the PATH for this one.

Done in the most recent commit.

Reply:
Thanks for setting the PATH!

Leave the check as-is then.